core/src/main/scala/org/apache/spark/Dependency.scala

```diff
@@ -110,6 +116,12 @@ class ShuffleDependency[K: ClassTag, V: ClassTag, C: ClassTag](
 
   def getMergerLocs: Seq[BlockManagerId] = mergerLocs
 
+  def setShuffleMergeFinalized(mergeFinalized: Boolean): Unit = {
```

Comment:
Is there a change to set `false`? If no, could we rename it to `markShuffleMergeFinalized()`?